Operational-Procedures/index.rst

```diff
@@ -1,24 +1,33 @@
-.. This is the label that can be used as for cross referencing in the given area
-.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hypens
-
+.. Review the README in this directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this file's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Do not delete the expected section title, reference or related comments provided from the template.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. This is the label that can be used as for cross referencing this file.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
 .. _Operational-Procedures:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
 
-#####################################
+######################
 Operational Procedures
-#####################################
+######################
+
+.. This section should provide a brief, top-level description of the page.
 
 .. note::
-    This is a template file that is associated with a template directory structure. This note will be deleted when
-    the section is properly populated
+    This is a template file that is associated with a template directory structure. This note should be deleted when the section is properly populated
 
 This page defines the different types of operational procedures.
 
-
 .. toctree::
     :maxdepth: 2
```

Comment:
i think we just want a max depth of 1 here?
The TOC looks funny because it's showing an actual procedure as well as the 3 classifications.

Reply:
Agreed, and it is appropriate given our current expectation that these areas treated separately. Done.